src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -511,38 +541,57 @@ protected Object clone() throws CloneNotSupportedException {
     }
 
     /**
-     * Ensures that the object referenced by the given reference remains
-     * <a href="package-summary.html#reachability"><em>strongly reachable</em></a>,
-     * regardless of any prior actions of the program that might otherwise cause
-     * the object to become unreachable; thus, the referenced object is not
+     * Ensures that the given object remains
+     * <a href="package-summary.html#reachability"><em>strongly reachable</em></a>.
+     * This reachability is assured regardless of any optimizing transformations
+     * the virtual machine may perform that might otherwise allow the object to
+     * become unreachable (see {@jls 12.6.1}). Thus, the given object is not
```

Comment:
I recommend replacing `(see {@jls 12.6.1})` with something like ` (see JLS {@jls 12.6.1})`, an idiom used elsewhere in core libs in formatting links into the JLS.